test/e2e/apps/cronjob.go

```diff
@@ -480,10 +490,7 @@ func ensureHistoryLimits(c clientset.Interface, ns string, cronJob *batchv1.Cron
 	jobs, err := c.BatchV1().Jobs(ns).List(context.TODO(), metav1.ListOptions{})
 	framework.ExpectNoError(err, "Failed to ensure a finished cronjob exists by listing jobs explicitly in namespace %s", ns)
 	activeJobs, finishedJobs := filterActiveJobs(jobs)
-	if len(finishedJobs) != 1 {
-		framework.Logf("Expected one finished job in namespace %s; activeJobs=%v; finishedJobs=%v", ns, activeJobs, finishedJobs)
-		framework.ExpectEqual(len(finishedJobs), 1)
-	}
+	gomega.Expect(len(finishedJobs)).To(gomega.BeNumerically("==", 1), fmt.Sprintf("Expected one finished job in namespace %s, got %d; activeJobs=%v; finishedJobs=%v", ns, len(finishedJobs), activeJobs, finishedJobs))
```

Comment:
Here the custom failure message includes `one` and the actual number, so `gomega.BeNumerically` is redundant.